test/azure/Expected/AcceptanceTests/AzureSpecials/azurespecialproperties/aio/operations_async/_subscription_in_credentials_operations_async.py

```diff
@@ -50,7 +51,7 @@ def __init__(self, client, config, serializer, deserializer) -> None:
         :param callable cls: A custom type or function that will be passed the direct response
         :return: None or the result of cls(response)
         :rtype: None
-        :raises: ~azurespecialproperties.models.ErrorException:
+        :raises: ~azure.mgmt.core.ARMError
```

Comment:
We also want to raise HttpResponseError, however we want to pass the kwarg "error_format" to "azure.core.mgmt.exceptions.ARMErrorFormat"

```
# now
raise ARMError(response=response, model=error)

# wanted
from azure.core.mgmt.exceptions import ARMErrorFormat
...
raise HttpResponseError(response=response, model=error, error_format=ARMErrorFormat)
```

Reply:
This applies to lambda as well if there is any